src/core/ButtonIcon/style.ts

```diff
@@ -152,6 +152,7 @@ const large = <ButtonIconSize extends keyof ButtonIconSizeToTypes>(
 
   return css`
     .MuiSvgIcon-root {
+      box-sizing: content-box;
```

Comment:
can we add a comment above to explain why we use content-box here instead of the default border-box please 🙏 Thanks so much!

Reply:
Sure, I'll add some 🤓

Reply:
Eew, we don't even need to adjust the box size if we simply replaced padding with margin. 🫠 My bad 😵‍💫